components/Badge/badge.html

```diff
@@ -20,16 +22,32 @@
     </nav>
     <main class="doc-container">
         <aside class="side-nav">
-            <h3>Get Started</h3>
-            <ul>
+            <div class="side-nav-header">
+                <h3>Get Started</h3>
+                <span id="get-started-expand-more" class="material-icons-outlined display-block" onclick="getStartedExpandMore()">
+                    expand_more
+                </span>
+                <span id="get-started-expand-less" class="material-icons-outlined display-none" onclick="getStartedExpandLess()">
+                    expand_less
+                </span>
+            </div>
+            <ul id="get-started-nav-items" class="get-started-nav-items">
                 <li class="side-nav-items">Avatar</li>
                 <li class="side-nav-items">Avatar2</li>
                 <li class="side-nav-items">Avatar3</li>
                 <li class="side-nav-items">Avatar4</li>
             </ul>
 
-            <h3>Components</h3>
-            <ul>
+            <div class="side-nav-header">
+                <h3>Components</h3>
+                <span id="components-expand-more" class="material-icons-outlined display-block" onclick="componentsExpandMore()">
+                    expand_more
+                </span>
+                <span id="components-expand-less" class="material-icons-outlined display-none" onclick="componentsExpandLess()">
+                    expand_less
+                </span>
+            </div>
+            <ul id="components-nav-items" class="components-nav-items">
                 <a href="../avatar/avatar.html"><li class="side-nav-items">Avatar</li></a>
                 <a href="../alert/alert.html"><li class="side-nav-items">Alert</li></a>
```

Comment:
Can we put the  a tag inside the li tag I think that would make more sense semantically since li tags comes inside ul but here it ul then a then li